test/Chainweb/Test/Pact/Utils.hs

```diff
@@ -397,14 +397,15 @@ testPactCtx v cid bhdb pdb = do
         t0 = BlockCreationTime $ Time (TimeSpan (Micros 0))
     ctx <- TestPactCtx
         <$> newMVar (PactServiceState Nothing mempty 0 t0 Nothing noSPVSupport)
-        <*> pure (PactServiceEnv Nothing cpe pdb bhdb (constGasModel 0) rs True 100000)
+        <*> pure (PactServiceEnv Nothing cpe pdb bhdb (constGasModel 0) rs True 100000 defaultOnFatalError)
```

Comment:
Can this use defaultPactServiceEnv instead

Reply:
ok